tests/Provider/YouTubeProviderTest.php

```diff
@@ -34,6 +35,9 @@
  */
 class YouTubeProviderTest extends AbstractProviderTest
 {
+    /**
+     * @param (MockObject&ClientInterface)|null $client
+     */
     public function getProvider(?object $client = null, ?RequestFactoryInterface $messageFactory = null): MediaProviderInterface
```

Comment:
As soon as you add a phpdoc to a method, psalm is not inheriting the phpdoc anymore.

So you have to add `@return YouTubeProvider`

Reply:
Fixed in other way